GoogleUtilities/Environment/third_party/GULAppEnvironmentUtil.m

```diff
@@ -209,6 +209,21 @@ + (BOOL)isSimulator {
   return NO;
 }
 
++ (NSString *)getSysctlEntry:(const char *)sysctlKey {
```

Comment:
Since all of these are static methods without much of context, can these be turned in as mere C functions? Any reason why they stay as static obj-c methods?

Reply:
They can be either - I chose to do a static method because it matched the rest of the class better, but no reason it can't be a C function.